### ISSUE TITLE
Edits should send typing notifications

### ISSUE THREAD

**Issue by turt2live** [vector-im/element-web#10188]
Maybe with custom status like https://github.com/matrix-org/matrix-doc/issues/437

**Comment by t3chguy** [comment]
Without custom status it'd be jarring if a user is editing a message older than the latest few as after the typing nothing would visibly show up for other users

**Comment by turt2live** [comment]
This could also be made possible with https://github.com/matrix-org/matrix-doc/pull/3038

ie: "Travis is editing a message"

**Comment by t3chguy** [comment]
That's what I assumed the original link was - I was wrong

**Comment by turt2live** [comment]
Original link was for which device rather than content - I repurposed it a bit wrongly some time ago though.

**Comment by ShadowJonathan** [comment]
Opposite of https://github.com/vector-im/element-web/issues/16675?

**Comment by SimonBrandner** [comment]
Yeah, this doesn't seem to be relevant anymore, scream if you disagree

**Comment by turt2live** [comment]
This is the feature request we should keep.
